src/api/config/application.rb

```diff
@@ -16,6 +16,7 @@ class Application < Rails::Application
     # Custom directories with classes and modules you want to be autoloadable.
     # config.autoload_paths += %W(#{config.root}/extras)
     config.eager_load_paths << Rails.root.join('lib', 'backend')
+    config.eager_load_paths << Rails.root.join('lib', 'crypto')
```

Comment:
Why do you want to eager load this for everybody? Why not require this in the only place you use it, the configuration model?